aiobotocore/utils.py

```diff
@@ -92,7 +92,7 @@ async def _fetch_metadata_token(self):
                         return None
                     elif response.status_code in (400,):
                         raise BadIMDSRequestError(request)
-                except ReadTimeoutError:
+                except (ReadTimeoutError, ConnectTimeoutError):
```

Comment:
this is probably actually an issue with our exception mapping here: https://github.com/aio-libs/aiobotocore/blob/master/aiobotocore/httpsession.py#L197 mind investigating?

Reply:
ya see: https://github.com/boto/botocore/blob/develop/botocore/utils.py#L461 I'm pretty sure we're just not mapping the exceptions correctly, we should probably port that relevant test over as well

Reply:
Now that you mention it, I believe you're right: https://docs.aiohttp.org/en/stable/client_reference.html?highlight=servertimeouterror#aiohttp.ServerTimeoutError

The aiohttp docs say that `ServerTimeoutError` is also for read timeouts! I'll update my PR to reflect that.

---------------

Also, wow, that was the fastest response I've ever gotten to a PR! Thanks:)

Reply:
Updated!

Reply:
Welcoming a little distraction from this issue I'm working on :) added reply